### PR TITLE
newshell - Add support for `pf` commands

### PR DESCRIPTION
**Pull request by ret2libc**
Add support for `pf` commands in newshell.
This is going to be a mess, unfortunately... `pf` commands have almost any characters inside. My 2cents is that they need to be rethought a bit, once this is done. For now I am trying to consider the arguments of `pf` as one single string, though that probably will require changes in tests.

For example, things like `(foo; pf (3xx))` will probably need to become `(foo; "pf (3xx)")` to make sure to distinguish the ending parenthesis `)` correctly.

### CODE REVIEW

**Comment by XVilka** [comment]
In theory, `pf` commands should be parsed by the new parser as well - it will help to solve the issues like nested `pf.xxx.yyy.zzz` autocompletion: https://github.com/radareorg/radare2/issues/6350#issue-196599016

@radare what do you think about changing `pf` syntax a bit while we're on it?

**Comment by ret2libc** [comment]
@XVilka do we have a full "specification" of the pf syntax? I can work on it.

**Comment by XVilka** [comment]
Not really, but most of the syntax should be covered in `pf?` and `pf??`

**Comment by ret2libc** [comment]
master: `71 XX`, this PR: `31 XX` yeah!

**Comment by ret2libc** [comment]
> master: `71 XX`, this PR: `31 XX` yeah!

Actually, PR: `24 XX`!

File: libr/core/cmd.c [hunk 1]
```diff
@@ -4889,7 +4908,7 @@ static bool substitute_args_do(struct tsr2cmd_state *state, RList *edits, TSNode
 static bool substitute_args(struct tsr2cmd_state *state, TSNode args, TSNode *new_command) {
 	RList *edits = r_list_newf ((RListFree)free_tsr2cmd_edit);
 
-	if (is_ts_args (args) || is_ts_arg (args)) {
+	if (is_ts_args (args) || is_ts_arg (args) || is_ts_pf_dot_cmd_args (args) || is_ts_pf_new_args (args) || is_ts_pf_args (args)) {
```
Comment (XVilka) [inline libr/core/cmd.c:4911]: Please split the long line

**Pull request merged**